examples/grafana-alloy-auto-instrumentation/ebpf/docker/docker-compose.yml

```
@@ -3,22 +3,20 @@ services:
     image: grafana/pyroscope
     ports:
     - 4040:4040
-  grafana-agent:
-    image: grafana/agent:main
+  grafana-alloy:
+    image: grafana/alloy:main
```

Maybe we should use a released version for alloy too?


```suggestion
    image: grafana/alloy:latest
```